content/en/developers/dogstatsd/_index.md

```diff
@@ -276,22 +281,26 @@ Or manually clone the repository at [github.com/DataDog/php-datadogstatsd][1] an
 
 
 [1]: https://github.com/DataDog/php-datadogstatsd#php-datadog-statsd-client
-{{% /tab %}}
-{{% tab ".NET" %}}
 
-[Get the package from NuGet][1] to install it.
+{{< /programming-lang >}}
 
+{{< programming-lang lang=".NET" >}}
+
+Get [the package from NuGet][1] to install it.
```

Comment:
No need to fix it in this PR, but it seems that url formating is off in code blocks: https://a.cl.ly/WnuBeeJ2 https://docs-staging.datadoghq.com/kari/codelang-switcher/developers/dogstatsd/?tab=hostagent#install-the-dogstatsd-client

Reply:
Yeah, I noticed that too. The work-around is to add other formatting. On other pages, I turned single <p> lines into <ul>s and then the <a> text is formatted correctly. Maybe I'll do this here too. But yeah, I'll also open a Web ticket. Thank you!